x-pack/plugins/cloud_security_posture/public/pages/configurations/latest_findings_by_resource/resource_findings/resource_findings_container.tsx

```diff
@@ -179,10 +166,10 @@ export const ResourceFindings = ({ dataView }: FindingsBaseProps) => {
       const newFindingIndex = nextFindingIndex % pageSize;
 
       // if the finding is not in the current page, we need to change the page
-      const pageIndex = Math.floor(nextFindingIndex / pageSize);
+      const flyoutPageIndex = Math.floor(nextFindingIndex / pageSize);
```

Comment:
Why did you name this code `flyoutPageIndex`?

Reply:
I needed a new name for the `pageIndex` variable here since this variable is already declared on top, so since we are calculating the page index for the flyout here, I thought it would be suitable to change it to `flyoutPageIndex`